### ISSUE TITLE
Uncaught (in promise)

### ISSUE THREAD

**Issue by tsirlucas** [vaneenige/rainbow-explorer#1]
Hey, just got this error on the [production link](https://use-the-platform.com/rainbow-explorer/) and seems like you're trying to get user's media before ask for permission. Is that right? Just reporting...

**Comment by vaneenige** [comment]
Thanks for the heads up! I've checked it and the issue seems to be "Unknown name of constraint detected - facingMode". It seems to be fine in Safari but Chrome is broken. I'll have a look!

**Comment by tsirlucas** [comment]
Yeah I just gave it all permissions and the error changed to "Unknown name of constraint detected - facingMode". Good luck! haha
